client/bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/dto/AcceptedSubmissionRequest.kt

```diff
@@ -0,0 +1,6 @@
+package ac.uk.ebi.biostd.client.dto
+
+data class AcceptedSubmissionRequest(
```

Comment:
I think this one need to be call AcceptedSubmissionResponse

Reply:
I called it request because that's what's created after the command is succesfully executed but yeah, it might be confusing. I'm even thinking we could just call it `AcceptedSubmission`